arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -308,7 +284,7 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK -- List of storage accounts in the given resource group retrieved and returned successfully.",
```

Comment:
How about `OK -- Storage accounts under the given resource group were retrieved successfully.`